### PR TITLE
Add a block of text to help with diff testing

### PR DESCRIPTION
**Pull request by MannyGrill**
Draft PR

test

### CODE REVIEW

**Review by GitKrakenQA — COMMENTED** [review]
This is a comment on a review

**Review by GitKrakenQA — CHANGES_REQUESTED** [review]
Can you address these changes, Manny?

**Comment by MannyGrill** [comment]
You got it

**Review by GitKrakenQA — APPROVED** [review]

**Review by GitKrakenQA — APPROVED** [review]

**Review by MannyGrill — COMMENTED** [review]
this is a test

File: blockOfText.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+And I see him jump before me, when I jump into my bed.
+
+The funniest thing about him is the way he likes to grow—
+Not at all like proper children, which is always very slow;
+For he sometimes shoots up taller like an india-rubber ball,
+And he sometimes gets so little that there’s none of him at all.
+
+---
+
+### A Snow Man
```
Comment (MannyGrill) [inline blockOfText.md:16]: code comment [pending]

**Review by MannyGrill — COMMENTED** [review]
comment

File: blockOfText.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+*Robert Louis Stevenson*
+
+I have a little shadow that goes in and out with me,
+And what can be the use of him is more than I can see.
+He is very, very like me from the heels up to the head;
+And I see him jump before me, when I jump into my bed.
+
+The funniest thing about him is the way he likes to grow—
+Not at all like proper children, which is always very slow;
+For he sometimes shoots up taller like an india-rubber ball,
```
Comment (MannyGrill) [inline blockOfText.md:11]: test

**Review by MannyGrill — COMMENTED** [review]
Finish review tags should go away

**Pull request closed without merging**